app/cdap/components/AddTransformation/SelectColumnsList/index.tsx

```diff
@@ -0,0 +1,247 @@
+                  classes={{
+                    head: `${classes.recipeStepsTableHeadStyles} ${classes.columnstyles}`,
+                  }}
+                ></TableCell>
+                <TableCell
+                  classes={{
+                    head: `${classes.recipeStepsTableHeadStyles} ${classes.nullValueHead}`,
+                  }}
+                >
+                  {T.translate('features.WranglerNewAddTransformation.columns')}
+                </TableCell>
+                <TableCell
+                  classes={{
+                    head: `${classes.recipeStepsTableHeadStyles} ${classes.nullValueHead}`,
+                  }}
+                >
+                  {T.translate('features.WranglerNewAddTransformation.nullValues')}
+                </TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {columns.map((eachColumn, index) => {
+                if (directiveFunctionSupportedDataType.includes('all')) {
+                  return (
+                    <TableRow
+                      className={`${classes.recipeStepsTableRowStyles} ${classes.rowsOfTable}`}
+                      key={index}
+                    >
+                      <TableCell
+                        classes={{
+                          body: `${classes.recipeStepsTableRowStyles} ${classes.radioButtonCellStyles}`,
+                        }}
+                      >
+                        <Radio
+                          color="primary"
+                          onChange={(e) => onSelect(eachColumn)}
+                          checked={
+                            selectedColumns.filter((el) => el.label == eachColumn.label).length
+                              ? true
+                              : false
+                          }
+                        />
+                      </TableCell>
+                      <TableCell classes={{ body: classes.recipeStepsTableRowStyles }}>
+                        <Typography className={classes.recipeStepsActionTypeStyles}>
+                          {eachColumn.label}
+                        </Typography>
+                        <Typography className={classes.recipeStepsActionTypeStyles}>
+                          {eachColumn.type}
+                        </Typography>
+                      </TableCell>
+                      <TableCell
+                        className={[
+                          classes.recipeStepsTableRowStyles,
+                          classes.circularBarCell,
+                        ].join(' ')}
+                      >
+                        {dataQualityValue?.length && (
+                          <DataQualityProgress value={dataQualityValue[index]?.value} />
+                        )}
+                      </TableCell>
+                    </TableRow>
+                  );
+                } else if (
+                  directiveFunctionSupportedDataType?.includes(eachColumn?.type[0]?.toLowerCase())
```

Comment:
dont write if else inside the component body, plz use conditional operations && and ternary ops ? :

Reply:
Done